File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int count=1;
        int n=nums.size();
        for(int i=1; i<n; i++){
            if(nums[i]!=nums[count-1]){
                nums[count++]=nums[i];
            }
        }
        return count;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        count=1;
        n=len(nums);
        for i in range(0,n):
            if nums[i]!=nums[count-1]:
                nums[count]=nums[i]
                count+=1
    
        return count;